mockclient.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
This file is obsolete, now isn't it?

Reply:
The "real" and mock HTTP client are in separate files. The types aren't exported so there's no "need" to.

Did you want me to merge `httpclient.go` and `mockclient.go` into `enviornment.go`?

Reply:
@JanDeDobbeleer ^ (Unsure if you get a notification when I comment on a PR)